Sources/ClientRuntime/Config/DefaultSDKRuntimeConfiguration.swift

```diff
@@ -94,7 +94,10 @@ public extension DefaultSDKRuntimeConfiguration {
         return URLSessionHTTPClient(httpClientConfiguration: httpClientConfiguration)
         #else
         let connectTimeoutMs = httpClientConfiguration.connectTimeout.map { UInt32($0 * 1_000_000) }
-        let config = CRTClientEngineConfig(connectTimeoutMs: connectTimeoutMs)
+        let config = CRTClientEngineConfig(
```

Comment:
The config object to which the CRT `logLevel` field was added is specifically to configure the CRT HTTP client.

Unless CRT only logs from the HTTP client (which I don't think is the case) then the CRT log level should be set elsewhere.

SRA has a Logging section, does it talk about logger configuration?

Reply:
The config exposed via HTTP client configuration gets used in CRT client engine initialization, where it re-assigns a logger to have the config log level value if the user-provided config value is anything other than `.none`. Given our SDK statically initializes `SDKDefaultIO` and shares it until program closes, we'd be missing CRT log from CRT resource initializations, but any CRT log after initialization would be logged properly.

Reply:
Remember that the CRT client engine is not always used.  In that case, the logger would not get configured at all, right?

Reply:
Hm... come to think of it - if the user doesn't use the CRT http client then log level won't be set to user-provided value, so for users on mac that use the URLSession client, CRT log would still be .none.... I think I do need to find a earlier place in program execution to allow user to configure log level for CRT.

Reply:
Changed the approach from exposing a config value to adding a setter.